zerver/lib/actions.py

```diff
@@ -2568,6 +2579,19 @@ def gather_subscriptions_helper(user_profile):
             subscribed.append(stream_dict)
         else:
             unsubscribed.append(stream_dict)
+    never_subscribed_stream_ids = [item for item in all_streams_id if item not in sub_unsub_stream_ids]
+    never_subscribed_streams = [x for x in all_streams if x['id'] in never_subscribed_stream_ids]
```

Comment:
Why are you using `x` for the variable name in this list comprehension? `stream` seems more natural here